src/FakeItEasy/ExceptionMessages.cs

```diff
@@ -57,6 +57,12 @@ internal static class ExceptionMessages
         public static string GetAssemblyFileNamesToScanForExtensionsShouldntReturnNull =>
             "IBootstrapper.GetAssemblyFileNamesToScanForExtensions must not return null";
 
+        public static string WrappingFakeCannotRaiseEvent =>
+            "The fake cannot raise the event because it's a wrapping fake and the event subscription is managed by the wrapped object. The event can only be raised internally by the wrapped object";

```

Comment:
nit: I think you're missing a "." at the end.

Reply:
Fixed (amended the commit)